src/environmentd/tests/sql.rs

```diff
@@ -3645,3 +3645,102 @@ async fn test_retain_history() {
         );
     }
 }
+
+#[mz_ore::test(tokio::test(flavor = "multi_thread", worker_threads = 1))]
+async fn test_temporal_static_queries() {
```

Comment:
Hm, does this have an advantage over using slt directly?

Reply:
The problem with creating an SLT test, is that the answer changes depending on the current time. I couldn't easily figure out a way to write an SLT test that wouldn't break in the future. Any ideas?

Reply:
Maybe we could change Jan's repro to make `mz_now()` comparisons only to dates very far in the future, e.g., in 2040 or something.

Reply:
Right, maybe testdrive with `$ set-regex` would help but that's fine then.